.gitmodules

```diff
@@ -34,3 +34,6 @@
 [submodule "plugins/sqs/aws-sdk-cpp"]
 	path = plugins/sqs/aws-sdk-cpp
 	url = https://github.com/hpcc-systems/aws-sdk-cpp
+[submodule "configuration/configmgr/RapidJSON"]
+        path = configuration/configmgr/RapidJSON
+        url = https://github.com/Tencent/rapidjson.git
```

Comment:
If submoduling from github, we prefer to fork and then submodule from our local copy (so that our builds don't break if the original repo is removed, renamed, force-pushed etc.)

Reply:
Also need to check the licence. In particular looks like we might need to include some acknowledgement that we use it to our binary releases?

What are we using it for that jptree doesn't already provide?

Reply:
One thing RapidJSON offers is JSON schema support so that modification templates don't have to be validated as part of parsing.

Reply:
Licence has been approved by legal, and I have forked the upstream repo. Can you change your PR so that it uses our fork for the submodule please.

Reply:
Also probably need to update our licence.txt to mention that we use this component, as per the terms of their licence.